tests/testthat/test-derive_params_growth_age.R

```diff
@@ -335,7 +335,7 @@ test_that("derive_params_growth_age Test 6: Test out of bound ages", {
     )
   )
 
-  expected <- c(NA_real_, NA_real_)
+  expected <- numeric()
```

Comment:
@Minlei0201 @rossfarrugia Can't quite wrapy my head around whether or not we really should be creating empty rows for unmatched records due to the nature of how the metadata needs to be split anyway, like in theory if you're running weight for age via who you'll have unmatched records for cdc and vice versa. I was running into issues trying to recreate david's dataset with the retention of unmatched records

Reply:
What are the pro and cons to create empty records? I don't think we need them, at least in the final dataset.
I had to manipulate David's dataset in order to compare the various results, I don't think that his dataset is in a proper shape, but we could imagine users having different metadata and resulting to such dataset.